src/DateField.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { ChronoDateCast, DateField } from './DateField'
+describe('Cast Function tests ->', () => {
+  const makeCastAssert = (castFn: any) => {
+    const assertFn = (raw: any, output: any): void => {
+      expect(castFn(raw)).toBe(output)
+    }
+    return assertFn
+  }
+  const makeCastAssertException = (castFn: any) => {
+    const assertFn = (raw: any, error: string): void => {
+      expect(() => {
+        castFn(raw)
+      }).toThrow(error)
+    }
+    return assertFn
+  }
+
+  const assertThrow = makeCastAssertException(ChronoDateCast)
+    const assertDC = (raw: any, output: any): void => {
+      expect(ChronoDateCast(raw)).toStrictEqual(output)
+    }
+
+
+  test('DateCast handles null cases', () => {
+    // eventually I want this 2500 test file from pandas to be our goal for DWIM date parsing
+    // https://github.com/pandas-dev/pandas/blob/main/pandas/tests/tools/test_to_datetime.py
+    assertDC(undefined, null)
+    assertDC(null, null)
+    assertDC('', null)
+  })
+
+  test('DateCast handles real dates', () => {
+    const dString = '2022-07-30'
+    const d = new Date(dString)
+    //assertDC(dString, d)  // this is the original datecast test, commented out because it doesn't work, the following does work
+
+    const dString2 = '2022-07-30T16:00:00.000Z'
+    const d2 = new Date(dString2)
+    assertDC(dString2, d2)
+  })
+
```

Comment:
@hansjhoffman @alnoorp can we discuss what we want to do for timezones and defaults?

Reply:
Default to UTC since the code is executed in AWS which likely wont be in their timezone?